bigquery/CHANGES.md

```diff
@@ -1,4 +1,4 @@
-### 1.2.1 - 2021-XX-XX
+### 1.2.1 - 2021-04-14
```

Comment:
You mean 2022?

Reply:
Not sure about previous dates as well.

Reply:
Do we have changes to bigquery connector ? We can skip releasing bigquery hadoop connector